Framework/PythonInterface/test/python/plugins/algorithms/WorkflowAlgorithms/IndirectILLReductionDIFFTest.py

```diff
@@ -0,0 +1,59 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2018 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+
+import unittest
+from mantid.simpleapi import mtd
+from mantid import config
+from testhelpers import run_algorithm
+from mantid.api import MatrixWorkspace
+
+
+class IndirectILLReductionDIFF(unittest.TestCase):
+    # cache the def instrument and data search dirs
+    facility = config['default.facility']
+    instrument = config['default.instrument']
+    data_dir = config['datasearch.directories']
+
+    # Doppler
+    doppler_run = '276047'
```

Comment:
Do we have enough data to make a test with multiple files to cover the conjoin and transpose aspects?
Potentially also with a composite algebra, like R1+R2, R3+R4.

Reply:
I only have one file of which I'm sure it comes from Doppler, but I could try and see if I find others